src/main/java/net/dv8tion/jda/core/requests/restaction/OrderAction.java

```diff
@@ -174,18 +350,49 @@ public M swapPosition(T swapEntity)
         return swapPosition(orderList.indexOf(swapEntity));
     }
 
+    /**
+     * Reverses the {@link #getCurrentOrder() current order} by using
+     * {@link java.util.Collections#reverse(java.util.List) Collections.reverse(orderList)}
+     *
+     * @return The current OrderAction sub-implementation instance
+     *
+     * @see    java.util.Collections#reverse(java.util.List)
+     */
     public M reverseOrder()
     {
         Collections.reverse(this.orderList);
         return (M) this;
     }
 
+    /**
+     * Shuffles the {@link #getCurrentOrder() current order} by using
+     * {@link java.util.Collections#shuffle(java.util.List) Collections.shuffle(orderList)}
+     *
+     * @return The current OrderAction sub-implementation instance
+     *
+     * @see    java.util.Collections#shuffle(java.util.List)
+     */
     public M shuffleOrder()
     {
         Collections.shuffle(this.orderList);
         return (M) this;
     }
 
+    /**
+     * Sorts the {@link #getCurrentOrder() current order} based on
+     * the specified {@link java.util.Comparator Comparator}.
+     * <br>Using {@link java.util.ArrayList#sort(java.util.Comparator) ArrayList.sort(comparator)}
```

Comment:
We should reference List#sort, not ArrayList. In the future we might chose to change to a different List implementation if we so chose.